scene/2d/gpu_particles_2d.cpp

```diff
@@ -141,7 +141,6 @@ void GPUParticles2D::set_process_material(const Ref<Material> &p_material) {
 void GPUParticles2D::set_trail_enabled(bool p_enabled) {
 	trail_enabled = p_enabled;
 	RS::get_singleton()->particles_set_trails(particles, trail_enabled, trail_length);
-	update_configuration_warnings();
```

Comment:
This one was copied from the 3D implementation which does have a warning, but there's no matching warning in 2D.